### PR TITLE
feat: LOPS-33: UI page to view listing of CandidateTasks in Dataset

### PR DESCRIPTION
**Pull request by yyassi-heartex**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: src/components/DataManager/Toolbar/Toolbar.js [hunk 1]
```diff
@@ -22,7 +23,7 @@ export const Toolbar = injector(observer(({ store }) => {
               return Instrument ? (
                 <Instrument
                   key={`instrument-${instrument}-${i}`}
-                  size="medium"
+                  size={isFF(FF_LOPS_E_3) ? "large" : "medium"}
```
Comment (bmartel) [inline src/components/DataManager/Toolbar/Toolbar.js:26]: Please update the label and review buttons on DataManager to use the correct size.

File: src/components/MainView/DataViewOld/Table.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const injector = inject(({ store }) => {
     total: dataStore?.total ?? 0,
     isLoading: dataStore?.loading ?? true,
     isLocked: currentView?.locked ?? false,
-    hasData: (store.project?.task_count ?? store.project?.task_number ?? 0) > 0,
+    hasData: (store.project?.task_count ?? store.project?.task_number ?? dataStore?.total ??0) > 0,
```
Comment (bmartel) [inline src/components/MainView/DataViewOld/Table.js:35]:
```suggestion
    hasData: (store.project?.task_count ?? store.project?.task_number ?? dataStore?.total ?? 0) > 0,
```

File: src/components/Common/SemanticSearch/SemanticSearch.js [hunk 3]
```diff
@@ -1,40 +1,131 @@
...
+              {!!currentValue?.length && (
+                <Elem name='icon' mod={{ delete: true }} onClick={clearHandler}>
+                  <LSPlus/>
+                </Elem>
+              )}
+              <Dropdown.Trigger
+                content={<SemanticSearchDropdown />}
+              >
+                <Elem name='icon' mod={{ filter: true }}>
+                  <RiFilter3Line size={18}/>
```
Comment (bmartel) [inline src/components/Common/SemanticSearch/SemanticSearch.js:121]:
```suggestion
                  <RiFilter3Line size={18} />
```

File: src/components/Common/SemanticSearch/SemanticSearch.js [hunk 4]
```diff
@@ -1,40 +1,131 @@
...
+              ref={inputRef}
+              placeholder={placeholder ?? `Search ${store?._sdk?.type === "DE" ? 'dataset' : 'project'}`}
+              onChange={onChangeHandler}
+              style={style}
+              {...(schema ?? {})}
+            />
+            <Elem name="controls">
+              {!!currentValue?.length && (
+                <Elem name='icon' mod={{ delete: true }} onClick={clearHandler}>
+                  <LSPlus/>
```
Comment (bmartel) [inline src/components/Common/SemanticSearch/SemanticSearch.js:114]:
```suggestion
                  <LSPlus />
```

File: src/components/DataManager/Toolbar/Toolbar.js (hunk 1 above)
Comment (yyassi-heartex) [reply src/components/DataManager/Toolbar/Toolbar.js:26]: done 
image

**Pull request merged**